RELEASE-NOTES.md

```diff
@@ -8,145 +8,160 @@ Instead update the vNext section until 4.0.0 is out.
+    - `pymc.sampling_jax` samplers support `log_likelihood`, `observed_data`, and `sample_stats` in returned InferenceData object (see [#5189](https://github.com/pymc-devs/pymc/pull/5189))
+    - Adding support for `pm.Deterministic` in `pymc.sampling_jax` (see [#5182](https://github.com/pymc-devs/pymc/pull/5182))
+
+- BART:
+  - Added partial dependence plots and individual conditional expectation plots [5091](https://github.com/pymc-devs/pymc3/pull/5091).
+  - Modify how particle weights are computed. This improves accuracy of the modeled function (see [5177](https://github.com/pymc-devs/pymc3/pull/5177)).
+  - Improve sampling, increase default number of particles [5229](https://github.com/pymc-devs/pymc3/pull/5229).
+  - A BART variable can be combined with other random variables. The `inv_link` argument has been removed (see [4914](https://github.com/pymc-devs/pymc3/pull/4914)).
+  - And many more improvements now in [pymc-experimental](https://www.pymc.io/projects/experimental/en/latest/index.html).
 
-### Unexpected breaking changes (action needed)
-+ New API is not available in `v3.11.5`.
-+ Old API does not work in `v4.0.0`.
+- Miscellaneous:
+  - The new `pm.find_constrained_prior` function can be used to find optimized prior parameters of a distribution under some
+  constraints (e.g lower and upper bound). See [#5231](https://github.com/pymc-devs/pymc/pull/5231).
+  - Nested models now inherit the parent model's coordinates. [#5344](https://github.com/pymc-devs/pymc/pull/5344)
+  - `softmax` and `log_softmax` functions added to `math` module (see [#5279](https://github.com/pymc-devs/pymc/pull/5279)).
+  - Added the low level `compile_forward_sampling_function` method to compile the aesara function responsible for generating forward samples (see [#5759](https://github.com/pymc-devs/pymc/pull/5759)).
 
-All of the above apply to:
+### Ongoing deprecations 🚧
+- Old API still works in `v4` and has a deprecation warning.
+- Preferably, the new API should be available in `v3` already.
 
-- ⚠ Random seeding behavior changed (see [#5787](https://github.com/pymc-devs/pymc/pull/5787))!
-    - Sampling results will differ from those of V3 when passing the same `random_seed` as before. They will be consistent across subsequent V4 releases unless mentioned otherwise.
-    - Sampling functions no longer respect user-specified global seeding! Always pass `random_seed` to ensure reproducible behavior.
-- Signature and default parameters changed for several distributions:
+This includes API changes we did not warn about since at least `3.11.0` (2021-01).
+
+- ⚠️ Setting initial values through `pm.Distribution(testval=...)` is now `pm.Distribution(initval=...)`.
+- ⚠️ Alternative `sd` keyword argument has been removed from all distributions. `sigma` should be used instead (see [#5583](https://github.com/pymc-devs/pymc/pull/5583)).
```

Comment:
Can we get rid of this section and send the individual items to other sections? Sounds confusing when I read it.

Reply:
Yeah, I don't like it either. Can probably just go to breaking changes

Reply:
Done